echo.go

```diff
@@ -338,6 +338,7 @@ var (
 	ErrCookieNotFound         = errors.New("cookie not found")
 	ErrInvalidCertOrKeyType   = errors.New("invalid cert or key type, must be string or []byte")
 	ErrInvalidListenerNetwork = errors.New("invalid listener network")
+	ErrEmptyContentType       = ErrUnsupportedMediaType.WithInternal(errors.New("missing content type header"))
```

Comment:
please do not add new public error - I do not think this case is something that you would actively check. By making it public instead of just returning `ErrUnsupportedMediaType.WithInternal(errors.New("missing content type header"))` we are making contract library users that there is logic somewhere that returns that error and we can not change it. This is not probably this kind of situation where we should make such strong commitment.

Reply:
Hi @aldas , Does this PR need some cleaning before becoming eligible for merge? I would like to take this, if possible.